mobile/lib/src/application/bluetooth/bluetooth_selector_state.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter_blue/flutter_blue.dart';
+
+abstract class BluetoothStates {}
+
+class BluetoothInitial extends BluetoothStates {}
+
+class BluetoothSearching extends BluetoothInitial {
```

Comment:
Pourquoi hériter de `BluetoothInitial` plutôt que `BluetoothStates` ?

Reply:
C'est un restant de quelque chose que j'avais essayé il y a longtemps, c'est plus nécessaire, switched back to BluetoothStates.